pkg/antctl/command_list_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Antrea Authors
+// Copyright 2022 Antrea Authors
```

Comment:
This is supposed to be the first year of the publication, so shouldn't be updated.

Reply:
fixed